src/main/java/duke/Task.java

```diff
@@ -0,0 +1,101 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Task represents a task by client.
+ * Task is an abstract class for Todo, Deadline and Event to inherit.
+ */
+abstract class Task {
+    protected String description;
+    protected boolean isDone;
+    protected String time;
+
+    /**
+     * Constructor of Task class.
+     *
+     * @param description description of the task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Second constructor of Task class.
+     *
+     * @param description description of the task.
+     * @param time time estimated to finish the task.
+     */
+    public Task(String description, String time) {
```

Comment:
Will it be clearer if the javadocs include when this second constructor could potentially be used to create tasks?

Reply:
I will add more details for this constructor in next commit 👍